uint8 masterID
uint8 control_mode
---
uint8 control_mode
